### PR TITLE
ARC pull request

### PR DESCRIPTION
**Pull request by samohvalovArt**

### CODE REVIEW

File: Factory/Classes/Facilities/Factory.m [hunk 1]
```diff
@@ -184,73 +166,73 @@ - (void)simulateWorkingDay
 
     NSError *error = nil;
```
Comment (Maxim-38RUS-Zabelin) [inline Factory/Classes/Facilities/Factory.m]: This line of code needs a small fix.

File: Factory/Classes/Facilities/Warehouse.m [hunk 2]
```diff
@@ -77,8 +77,8 @@ - (NSSet *)shipWaresOfCount:(NSUInteger)count
         for (id<WareProtocol> ware in mutableShipment) {
             [self.wares removeObjectForKey:[ware uniqueIdentifier]];
         }
-
-        return [mutableShipment copy];
```
Comment (Maxim-38RUS-Zabelin) [inline Factory/Classes/Facilities/Warehouse.m:81]: Actually, that was correct.

File: Factory/Classes/People/Person.h [hunk 3]
```diff
@@ -25,7 +25,7 @@
 /**
  *  Returns the current location of the person.
  */
-@property (nonatomic, readonly) id<LocationProtocol> location;
```
Comment (Maxim-38RUS-Zabelin) [inline Factory/Classes/People/Person.h:28]: This was also correct.

**Comment by Maxim-38RUS-Zabelin** [comment]
There are still some more things that need handling. Please find them and correct.

**Comment by samohvalovArt** [comment]
Ok
22 февр. 2015 г. 18:06 пользователь "Maxim Zabelin" <
notifications@github.com> написал:

**Comment by Maxim-38RUS-Zabelin** [comment]
There are even more mistakes than it was before...

**Pull request closed without merging**